srcpkgs/kdevelop/template

```diff
@@ -24,6 +24,11 @@ if [ "$XBPS_TARGET_ENDIAN" = "le" ]; then
 	build_options_default="webengine"
 fi
 
+case "$XBPS_TARGET_MACHINE" in
+	# qt5-webengine-devel unavailable
+	armv*) build_options_default="" ;;
+esac
+
 if [ "$build_option_webengine" ]; then
 	makedepends+=" qt5-webengine-devel"
```

Comment:
please put that into `makedepends=` and guard with `vopt_if()` like the manual recommends.